### PR TITLE
Frontend README update to add PR checks list

### PR DESCRIPTION
**Pull request by JackDawson94**
Got frustrated that there is no PR doc for people who only contribute to frontend.
So I added a few lines to the README

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -56,5 +56,29 @@ yarn format
...
+Run the code formatter:
+
+```shell
+yarn format
+```
+
+Run the typescript compiler checks:
+
+```shell
+tsc --noEmit
```
Comment (mmenanno) [inline frontend/README.md]: Should there be a note added as to where this comes from? Running this on my mac I get:
`zsh: command not found: tsc`

Comment (JackDawson94) [reply frontend/README.md]: Yeah I agree, I'm not sure how to address it. It comes with installing Typescript. Which isn't actually a dependency of the UI project.
So it's only necessary to run tsc...

Comment (peolic) [reply frontend/README.md]: > It comes with installing Typescript. Which isn't actually a dependency of the UI project.

[It is a `devDependency`](https://github.com/stashapp/stash-box/blob/bae0400549c51ec055920ea326ed736ba8f12987/frontend/package.json#L53), and the solution should be `yarn tsc --noEmit` or creating a separate entry in `scripts` for it.

**Comment by JackDawson94** [comment]
Updated based on your comments.

All the commands are packaged as "yarn validate" already. So I just added this to the Readme.

**Pull request merged**